Mechanics/Ship.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SpaceOpera.Mechanics
+{
+    class Ship
+    {
+        string name;
+        int baseHp;
+        int modifiedHp;
+        int currentHp;
```

Comment:
Perhaps create a structure or something that holds all three?
Something like:
Value {
OriginalValue;
ModifiedValue;
CurrentValue;
}